samples/musicfestival-frontend-react/src/pages/SearchPage.tsx

```diff
@@ -35,6 +44,7 @@ function SearchPage() {
     let queryString = searchParams.get("q") ?? ""
     let filterQueryString = searchParams.get("f") ?? ""
     const [filterBy, setFilterBy] = useState(filterQueryString ?? ARTIST)
+    const ranking = Ranking[capitalize(searchParams.get("r")?.toString().toLowerCase() || "") as keyof typeof Ranking] || Ranking.Relevance;
```

Comment:
Ranking like BoostOnly will not work with this logic to lowercase the whole parameter and capitalize the first letter (it will become 'Boostonly').  
As discussed, maybe it's better to loop through each enum values and compare with the query string parameter, ignoring case.  
The capitalize function if not used can be removed.